src/common/variables.scss

```diff
@@ -129,8 +150,12 @@ $cartTextColor:#686868;
 		}
   }
   
-
 }
+
+//-------------------------------------------------------------------------------------------------animation
+
+//-------------------------------------------------------HEADER
+
```

Comment:
IF you decide to add comments, be consistent with the styling : 

 `HEADER` and `animation` and 2 different things
 
 Either `HEADER ANIMATIONS` or `header animation`